docs/platform-setup.md

```diff
@@ -18,3 +18,124 @@ For each job, disable the GitHub multibranch status:
 - Select "Disable GitHub Notifications".
 
 - Click on "Save".
+
+## Deploy Nexus Docker Registry
+
+### Prerequisite
+
+- Nexus is installed.
+- NGINX Ingress Controller is installed.
+
+Deploying docker registry with nexus should follow this steps:
+
+### Create New Docker Repository
+
+Open the platform team's [Nexus](https://nexus.platform.dev.nuxeo.com/). Start new session using admin user
+
+#### Blop Storage And Repository Configuration
+
+- Create a blob storage : `docker-registry`
+
+  - In Server Administration and Configuration > Blob Stores,  click on the "Create blob store" button
+
+  - Complete all required fields and click on "Save"
+
+- Create a docker (hosted) nexus repository: `docker-registry`
+
+  - In Server Administration and Configuration > Repositories,  click on the "Create repository" button
+
+  - Complete all required fields and click on "Save"
```

Comment:
Shouldn't we mention at some point to use the previously created blob store?

Reply:
will be taken to account